WcaOnRails/app/assets/stylesheets/wca.scss

```diff
@@ -222,3 +222,26 @@ table.wca-results {
     padding: 10px;
   }
 }
+
+
+// Set flag imitations for competitions taking place in multiple countries.
+.f16.flag {
+  &.xe, &.xa, &.xs {
```

Comment:
Ah, darn. Our css style checker requires that each of these be on a separate line.